pr_agent/git_providers/bitbucket_provider.py

```diff
@@ -228,6 +231,26 @@ def _parse_pr_url(pr_url: str) -> Tuple[str, int]:
             raise ValueError("Unable to convert PR number to integer") from e
 
         return workspace_slug, repo_slug, pr_number
+    
+    @staticmethod
+    def _parse_issue_url(issue_url: str) -> Tuple[str, int]:
+        parsed_url = urlparse(issue_url)
+
+        if "bitbucket.org" not in parsed_url.netloc:
+            raise ValueError("The provided URL is not a valid Bitbucket URL")
+
+        path_parts = parsed_url.path.strip('/').split('/')
+        if len(path_parts) < 5 or path_parts[2] != "issues":
+            raise ValueError("The provided URL does not appear to be a Bitbucket issue URL")
```

Comment:
**Suggestion:** Consider handling the case where the Bitbucket issue URL is not valid. Currently, the code raises a ValueError, but it might be more user-friendly to log an error message and continue execution if possible. [enhancement]
```suggestion
        if "bitbucket.org" not in parsed_url.netloc:
            get_logger().error("The provided URL is not a valid Bitbucket URL")
            return None

        path_parts = parsed_url.path.strip('/').split('/')
        if len(path_parts) < 5 or path_parts[2] != "issues":
            get_logger().error("The provided URL does not appear to be a Bitbucket issue URL")
            return None
```